pallets/delegation/src/benchmarking.rs

```diff
@@ -26,7 +26,7 @@ use frame_support::{
 use frame_system::RawOrigin;
 use sp_core::{offchain::KeyTypeId, sr25519};
 use sp_io::crypto::sr25519_generate;
-use sp_runtime::MultiSignature;
+use sp_runtime::{traits::Zero, MultiSignature};
```

Comment:
This will create unnecessary conflicts with other PRs. IMHO you should try to change as little as possible. This will reduce the conflicts and thus saves us time.